feature-all-markup

Shtrohaim opened the pull request:
Готовый сайт с мобильной и desktop версией.

Shtrohaim committed:
Сделан первый блок welcome, а также header с burger menu

Shtrohaim committed:
Сделан блок sightseens, доделана навигация, исправлены некоторые элементы welcome

Shtrohaim committed:
Сделан блок contacts и footer, добавлено лого в header, исправлены некоторые недочёты в других блоках

Shtrohaim committed:
Сделан последний блок tours, исправлены недочёты прошлых блоков

Shtrohaim committed:
Исправлен баг с footer

Shtrohaim committed:
Убраны выделения ссылок на телефоне

Shtrohaim committed:
тестовый

Shtrohaim committed:
сделана desktop версия, доработана мобильная версия

Shtrohaim committed:
переработано меню добавлена анимация для бургера

Shtrohaim committed:
исправлен баг с при загрузке навигации

Shtrohaim committed:
исправлен html

Shtrohaim committed:
исправлен html №2

Shtrohaim committed:
Merge branch 'main' into feature-desktop-markup

MikeBazhenov commented on index.html:
```diff
@@ -1,9 +1,210 @@
+          </div>
+        </div>
+      </header>
+        <div class="sections">
```
`<main class="sections"`

Shtrohaim replied:
исправил

MikeBazhenov commented on index.html:
```diff
@@ -1,9 +1,210 @@
+            <div class="welcome__wrapper">
+              <h1 class="welcome__title">ХАКАСИЯ</h1>
+              <h4 class="welcome__description">Туристическое Агенство Республики Хакасия</h4>
+              <a class="welcome__arrowLink" href='#sightseens'>
```
a нужно заменить на <button type="button"

Shtrohaim replied:
заменил

MikeBazhenov commented on index.html:
```diff
@@ -1,9 +1,210 @@
+              <h1 class="welcome__title">ХАКАСИЯ</h1>
+              <h4 class="welcome__description">Туристическое Агенство Республики Хакасия</h4>
+              <a class="welcome__arrowLink" href='#sightseens'>
+                <span class="welcome__followArrow"></span>
```
Можно сверстать без span внутри

Shtrohaim replied:
убрал span

MikeBazhenov commented on index.html:
```diff
@@ -1,9 +1,210 @@
+          <section class="sightseens" id="sightseens">
+            <h2 class="sightseens__title">Достопримечательности</h2>
+            <div class="sightseens__wrapper">
+              <div class="firstSight">
```
class не по бэм

Shtrohaim replied:
исправил

MikeBazhenov commented on index.html:
```diff
@@ -1,9 +1,210 @@
+          <section class="tours" id="tours">
+            <h2 class="tours__title">Туры</h2>
+            <div class="tours__wrapper">
+              <div class="tour1">
```
Класс не по бэм

Shtrohaim replied:
исправил

MikeBazhenov commented on js/app.js:
```diff
@@ -0,0 +1,184 @@
+const openMenuButton = document.querySelector('.header__menuBurger');
```
Что будет если js скрипт подключить в head документа?

Shtrohaim replied:
Скрипт выполниться раньше прорисовки страницы,  поэтому кнопки и другие элементы он не найдёт и работать не будет. Но я переделал и теперь оно работает в head.

Shtrohaim committed:
div заменён на main

Shtrohaim committed:
bash: строка 3: M-\: команда не найдена
Merge branch 'feature-desktop-markup' of https://github.com/Shtrohaim/frontend-task-2 into feature-desktop-markup

Shtrohaim committed:
исправлены замечания с ссылкой и отрисовкой стрелки, БЭМом и скриптом

MikeBazhenov commented on index.html:
```diff
@@ -1,9 +1,206 @@
+          </section>
+          <section class="sightseens" id="sightseens">
+            <h2 class="sightseens__title">Достопримечательности</h2>
+              <div class="firstSight">
```
firstSight, secondSight, thirdSight

1. Нельзя использовать в названия class большие буквы
2. Блок достопримечательностей универсальный, их может быть и 20. Что делать если сервер вернет нам больше чем 3 достпримечательности? Как будет классы подстраивать?

Shtrohaim replied:
в документации БЭМ вроде как написано, что можно использовать camelCase

MikeBazhenov replied:
https://ru.bem.info/methodology/naming-convention/

Shtrohaim replied:
Понял, а со вторым пунктом мне надо, чтобы у них одно название у классов было?

MikeBazhenov replied:
Да одно название должно быть.

MikeBazhenov commented on index.html:
```diff
@@ -1,9 +1,206 @@
+            <h2 class="tours__title">Туры</h2>
+            <div class="allTours">
+              <div class="tour1">
+                <div class="tour1__allInfo">
```
Тоже самое как и с достопримечательностями

Shtrohaim replied:
Ну вроде как сделал

Shtrohaim committed:
исправлены замечания по турам, достопримичательностям и вообще по всему

MikeBazhenov commented on index.html:
```diff
@@ -1,9 +1,202 @@
+                <img class="sightseen__image" src="img/DJI_0185-Pano_1.jpg" alt='ts' />
+                <div class="sightseen__info" id="info3">
+                  <h3 class="sightseen__title">Сундуки</h3>
+                  <h4 class="sightseen__description1">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur pretium orci at rutrum pellentesque. Vivamus id nunc at ante vulputate egestas. Sed varius elit ornare condimentum fermentum. Integer mattis pretium tortor dictum tristique. Donec in congue mauris. Quisque vitae nisi ultricies, efficitur odio vel, porta nulla. Donec porta, maur ...
```
h4 заменить нужно на p
Это обычный текст, не заголовок

Shtrohaim replied:
Изменил

MikeBazhenov commented on index.html:
```diff
@@ -1,9 +1,202 @@
+                </div>
+              </div>
+            </div>
+              <div class="tours__show-more">Больше туров</div>
```
Посмотри как ребята на практике сделали
https://github.com/AJC667/AAAAAAAAAAAAAAAA

Shtrohaim replied:
Исправил

Shtrohaim committed:
исправлена кнопка показа туров и заголовки

MikeBazhenov commented:
Всё готово, можно мержить!
Ссылку на этот репозиторий отправить нужно в электронный курс в ответ на задание 2
